docs/DEPLOY.md

```diff
@@ -0,0 +1,53 @@
+
+## Deploy
+Deployment of team web on team10-20.studenti.fiit.stuba.sk.
```

Comment:
We could add the link in the link markdown syntax, couldn't we?
Ref: https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#links